yelp_beans/logic/meeting_spec.py

```diff
@@ -55,7 +55,14 @@ def get_users_from_spec(meeting_spec):
     return users
 
 
-def get_meeting_datetime(meeting_spec):
+def get_meeting_datetime(meeting_spec, user=None):
+    ''' Get the meeting datetime for user. If user is specified, the timezone will be the user's
+    timezone preference. If not specified, the timezone will be the meeting spec's timezone.
+    '''
+    if user and user.timezone:
```

Comment:
We should version beans - that way you can't run this code without also picking up the changes to the User model